compositor_pipeline/src/pipeline/decoder/opus.rs

```diff
@@ -71,21 +62,27 @@ fn run_decoder_thread(
                 }
             };
 
-        let mut decoded_samples = Vec::with_capacity(decoded_samples_count / 2);
+        let mut left = Vec::with_capacity(decoded_samples_count / 2);
+        let mut right = Vec::with_capacity(decoded_samples_count / 2);
         for i in 0..decoded_samples_count {
-            decoded_samples.push((buffer[2 * i], buffer[2 * i + 1]));
+            left.push(buffer[2 * i]);
+            right.push(buffer[2 * i + 1]);
```

Comment:
Are `left` and `right` capacities correct? It looks like you iterate over every decoded sample

Reply:
No. Moreover, this is useless. I forgot to remove it